config/environments/development.rb

```diff
@@ -22,8 +22,9 @@
   if Rails.root.join("tmp/caching-dev.txt").exist?
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
+    config.action_controller.page_cache_directory = Rails.public_path
 
-    config.cache_store = :memory_store
+    config.cache_store = :file_store, Rails.root.join("tmp/cache")
```

Comment:
Why is a file cache better than a memory cache? You say it gives better performance but normally memory is faster than than something that is disk backed? Bearing in mind that this affects all users is the performance benefit in fact only when running under docker?